### DISCUSSION TITLE
jspsych on mobile devices

### DISCUSSION THREAD

**Discussion by tmalsburg** [jspsych/jsPsych#2531] [Q&A] [answered]
I'd like to create experiments that work more or less equally well on desktops and mobile devices (phones, tablets). Mainly relatively simple surveys, multiple choice questions, likert scale ratings, stuff like that.  Is jspsych going to be a suitable platform for that?  I've tried the demo experiment from the documentation (simple RT task) and, out of the box, that didn't work on my phone (tiny fonts, keyboard doesn't appear when it's needed).

**Comment by Nega100** [comment, marked as answer]
Hi @tmalsburg  
I know this is not the best answer but I've tried jspsych using mobile devices some time ago. All went well but some displays will be messed up when the rotate screen is active. I'm using wordpress and inserting links for cognitive tasks using jspsych. then try it with a browser on mobile devices. For jspsych-html-keyboard response, it can be replaced using jspsych-html-button-response.
